### PR TITLE
Conditionally execute tests matching the versions of the cluster

### PR DESCRIPTION
**Pull request by jhou1**
@liangxia PTAL

### CODE REVIEW

File: ci-operator/step-registry/cucushift/pre/cucushift-pre-commands.sh [hunk 1]
```diff
@@ -65,14 +65,14 @@ runtime_env=${SHARED_DIR}/runtime_env
...
+export TAG_VERSION="@${ver_cli}"
+
 export CUCUMBER_PUBLISH_QUIET=true
 export DISABLE_WAIT_PRINT=true
 
-# Export those parameters before running
-export BUSHSLICER_DEFAULT_ENVIRONMENT=ocp4
 export OPENSHIFT_ENV_OCP4_HOSTS="${hosts}:lb"
 export OPENSHIFT_ENV_OCP4_ADMIN_CREDS_SPEC=file:///tmp/kubeconfig
+export BUSHSLICER_DEFAULT_ENVIRONMENT=ocp4
```
Comment (liangxia) [inline ci-operator/step-registry/cucushift/pre/cucushift-pre-commands.sh:76]: We need to set `BUSHSLICER_DEFAULT_ENVIRONMENT` first, the `OCP4` in `OPENSHIFT_ENV_OCP4_HOSTS` and `OPENSHIFT_ENV_OCP4_ADMIN_CREDS_SPEC` are coming from the value of this var.

Comment (jhou1) [reply ci-operator/step-registry/cucushift/pre/cucushift-pre-commands.sh]: Thank you! fixed now.

**Pull request merged**